scalafix/build.sbt

```diff
@@ -0,0 +1,60 @@
+lazy val V = _root_.scalafix.sbt.BuildInfo
+inThisBuild(
+  List(
+    organization := "com.example",
+    homepage := Some(url("https://github.com/http4s/http4s")),
+    licenses := List("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")),
+    developers := List(
+      Developer(
```

Comment:
About this: I actually don't care about being marked as the developer or anything. So please change it to work with your release process :)

Reply:
This doesn't touch the main build.sbt at all.  Would we deploy this as a submodule, or is it just a project-in-a-project?

Reply:
What you prefer, I didn't think about deployment at all and I don't know if it's best as a submodule or a project in a project. Your call :)